todoman/cli.py

```diff
@@ -194,13 +197,7 @@ def formatter(self):
 
 
 @click.group(invoke_without_command=True)
-@click.option(
-    '-v',
-    '--verbosity',
-    type=click.Choice(['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG']),
-    help='Set verbosity to the given level.',
-    default='WARNING',
-)
+@click_log.simple_verbosity_option(None)
```

Comment:
Explicit parameter should not be necessary now, right?

Reply:
Good catch, thanks!